src/components/Layout/Views/UserProfileView.vue

```diff
@@ -57,12 +57,27 @@
       </Boton>
     </div>
   </div>
+  <div
+    style="justify-content: center; display: flex;"
+    @click="deleteAccountMessage">
+    <Boton
+      type="rounded-red"
+      class="ocial-button"
+      style="width: 80%; padding-top: 1vh; padding-bottom: 1vh;">
```

Comment:
```suggestion
    >
    <Boton
      type="rounded-red"
      style="width: 80%; padding-top: 1vh; padding-bottom: 1vh;"
      @click="deleteAccountMessage">
```

Reply:
Otros @click están definidos en el div, no en el Boton. Entiendo que vaya en el Boton, pero entonces hay otros que están mal. Por ejemplo, en el logout de `UserProfileView`:

![image](https://github.com/ispp-2324-ocial/frontend/assets/80246400/68534860-64dc-4b2d-add5-3441da955530)


Además, veo que has borrado la clase `ocial-button`. No tiene ningún uso para definir css, pero en el proyecto hay varios botones definidos con esta clase y entiendo que se ha hecho para identificarlos:

![image](https://github.com/ispp-2324-ocial/frontend/assets/80246400/14ceb441-14fe-4883-adb4-63b5afc99112)

Reply:
Siendo estrictos, funcionalmente da igual donde vayan puesto que además [JavaScript desencadena los objetos para todo el árbol DOM de forma recursiva](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Events#event_bubbling). Si te fijas, da la casualidad de que en MDN aparece el mismo ejemplo que tenemos entre las manos.

Hay veces que el bubbling puede ser confuso de debuggear (por ejemplo, en un formulario no funcionaría el enter en este caso, puesto que el click debe de triggearse desde un ``button``). Por exactitud (lo que queremos que tire el evento es el botón, no el padre) y por claridad (más fácil de encontrar en el código donde se lanzan los eventos cuanto más preciso sea el elemento que realice la acción, en vez de si lo realizamos 3 capas más arriba) creo que es mejor que esté en el botón. 

Sobre las clases, realmente no creo que haya ninguna razón profunda, mas que ha habido muchas partes de la aplicación que se han ido construyendo haciendo copy/paste y este tipo de detalles (que en realidad son tonterías, pero al final no deja de ser código inútil, que aunque afecte muy insignificantemente, algo empeora el rendimiento durante la ejecución) pues se han ido pasando en las consiguientes revisiones.

Si quieres haz una PR eliminando dichas clases del resto de elementos (si es que no se usan, que igual en alguno de esos componentes si se les da uso!)